modules/combined/tests/creep_plasticity_combined/combined_stress_relaxation_sm.i

```diff
@@ -13,19 +13,22 @@
 #    stress_yy = |--------------------|
 #                \ 3*a*E^4*ef^3*t + 1 /
 #
-#    where E  = 2.8e7 (Young's modulus)
-#          a  = 3e-26 (creep coefficient)
-#          ef = 0.01  (displacement)
-#          t  =       (time)
+#    where E  = 2.0e11  (Young's modulus)
+#          a  = 3e-26  (creep coefficient)
+#          ef = 0.01   (displacement)
+#          t  = 2160.0    (time)
 #
-# The solution computed is very close to the exact solution.  This test is not a
-#     correct representation of the problem since it does not set the initial
-#     condition for the displacement and the stress.  Currently (1 March 2011),
-#     no capability exists for setting the initial condition of stress.  Until
-#     that is available, some error will be inherent in the solution.
+#    such that the analytical solution is computed to be 2.9518e3 Pa
```

Comment:
I'd like to talk for a bit about this.  If what was deleted in that comment is still true, we should probably leave that comment.